docs/api/dialog.md

```diff
@@ -187,8 +187,8 @@ and no GUI dialog will appear.
 Displays a modal dialog that shows a message and certificate information, and
 gives the user the option of trusting/importing the certificate.
 
-The `browserWindow` argument allows the dialog to attach itself to a parent
-window, making it modal.
+On macOS the `browserWindow` argument allows the dialog to attach itself to a parent
+window, making it modal. On Windows a modal dialog is not supported.
```

Comment:
This is probably getting way into obscure Win32 territory, but there's no way to pass a window handle to `CertAddCertificateContextToStore` to associate the popup message with a parent window - so modality isn't enforced here (that is, you can interact with the window without acknowledging the popup).

I need to tidy this these docs, but I also had a idea to show a dialog before this step with the `message` argument that the caller provides.